identity-credential/README.md

```diff
@@ -1 +1,46 @@
-Types and traits for working with Verifiable Credentials/Presentations.
+# Iota Identity: Credentials 
+
+This crate contains types representing *credentials* and *presentations* defined in the [W3C VC-Data model]((https://www.w3.org/TR/vc-data-model/). An overview of these concepts and how to work with them in the context of the Iota Identity Framework can be found [in the wiki](https://wiki.iota.org/identity.rs/concepts/verifiable_credentials/overview). 
+
+
+# Example
+Constructing a `Credential` using the `CredentialBuilder`. 
+
+```rust 
+  use identity_credential::credential::{Credential, CredentialBuilder, Subject, Issuer};
+  use serde_json::json;
+  use serde_json::Value;
+
+  // Construct a `Subject` from json
+  let json_subject: Value = json!({
+    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
+    "degree": {
+      "type": "BachelorDegree",
+      "name": "Bachelor of Science and Arts"
+    }
+  });
+  let subject: Subject = serde_json::from_value(json_subject).unwrap();
+
+  // Construct an `Issuer` from json
+  let json_issuer: Value = json!({
+    "id": "did:example:76e12ec712ebc6f1c221ebfeb1f",
+    "name": "Example University"
+  });
+
+  let issuer: Issuer = serde_json::from_value(json_issuer).unwrap();
+
+  let credential: Credential = CredentialBuilder::default()
+    .context(Url::parse("https://www.w3.org/2018/credentials/examples/v1").unwrap())
+    .id(Url::parse("http://example.edu/credentials/3732").unwrap())
+    .type_("UniversityDegreeCredential")
+    .subject(subject)
+    .issuer(issuer)
+    .issuance_date(Timestamp::parse("2010-01-01T00:00:00Z").unwrap())
+    .build()
+    .unwrap();
+
+```
+# Important 
+Although the `CredentialBuilder` generates a `Credential` it cannot be considered a *verifiable credential* until it has been signed by the issuer. See [this example](https://github.com/iotaledger/identity.rs/blob/support/v0.5/examples/account/create_vc.rs) for a full example explaining how to create a credential with the corresponding issuer's signature. 
```

Comment:
Should we use a relative link for the example, instead of pinning to the `0.5` version in this case?

Reply:
Not sure how that will work on `crates.io` though.

Reply:
oh yeah, good point. Good chance it won't work. Shall we still try with the initial release?

Reply:
Regarding relative links for examples, looks like it might be supported?

- https://www.reddit.com/r/rust/comments/rnlf38/relative_links_in_readme_files_on_cratesio/
- https://github.com/rust-lang/crates.io/issues/3484

Reply:
I think the examples should link properly now on crates.io at least if I understood this correctly: https://github.com/rust-lang/crates.io/pull/3861. On the other hand I cannot get the links to work when compiling locally with `$/identity-credential/ cargo doc --no-deps --open`. Is it possible to fix that? @cycraig @PhilippGackstatter @HenriqueNogara.

Reply:
Maybe because the `examples` directory is above the `identity-credential` crate level? I say we remove those links then?

Reply:
I think the issue is that `cargo doc` compiles the generated documentation to `target/doc`. Don't know how to get around this. I would prefer to link to full examples, as people may get the wrong idea that one can sign/verify with this crate as well.

Reply:
On the other hand we do link out to the Wiki at the start of this README so could be fine to just drop those links indeed.

Reply:
I decided on just removing the links for now. If it becomes a FAQ how to sign credentials/presentations we can deal with adding links to relevant examples/crates then.